.github/workflows/reusable_multi_numa.yml

```diff
@@ -1,4 +1,4 @@
-# Runs tests on multi-numa machine
+# Runs tests on multi-numa machines1
 name: MultiNuma
 
```

Comment:
pls squash all your commits into a single one, with a proper description what this commit does